operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,93 @@
+:_content-type: ASSEMBLY
+[id="authentication-provider-structure"]
+= Authentication provider structure
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+An authentication provider connects to a third-party source of user identity, for example, an identity provider or IDP, gets the user identity, issues a token based on that identity, and returns the token to RHACS. 
+This token allows RHACS to authorize the user. 
+RHACS uses the token within the user interface and API calls.
+
+After installing RHACS, you must set up your IDP to authorize users. 
```

Comment:
```suggestion
After installing RHACS, you must set up your idP to authorize users. 
```

Reply:
@agantony @ivan-degtiarenko - IDP is correct per Red Hat Supplementary Style Guide.